tests/integrated/test-naulin-laplace/test_naulin_laplace.cxx

```diff
@@ -43,7 +43,7 @@ int main(int argc, char** argv) {
 
   //class Laplacian* invert = Laplacian::create();
   Options* options = Options::getRoot()->getSection("laplace");
-  LaplaceNaulin* invert = new LaplaceNaulin(options);
+  auto* invert = new LaplaceNaulin(options);
```

Comment:
`unique_ptr`?

Reply:
I fixed this slightly differently... I added `LaplaceNaulin::resetMeanIterations`, which means we can  then just make the `LaplaceNaulin` object on the stack in this test and avoid the need for repeated new/deletes.

Possibly a little gross, but probably ok